go-controller/hybrid-overlay/pkg/controller/master_test.go

```diff
@@ -135,12 +150,15 @@ var _ = Describe("Hybrid SDN Master Operations", func() {
 			})
 
 			fexec := ovntest.NewFakeExec()
-			addGetPortAddressesCmds(fexec, nodeName, nodeHOMAC, nodeHOIP)
 
 			err := util.SetExec(fexec)
 			Expect(err).NotTo(HaveOccurred())
 			_, err = config.InitConfig(ctx, fexec, nil)
 			Expect(err).NotTo(HaveOccurred())
+			err = mock.InitMockOVNClients()
+			Expect(err).NotTo(HaveOccurred())
+			mockPortDelCmds(nodeName)
```

Comment:
we shouldn't be mocking deletes to clean things up, we should be starting with a fresh client for each test

Reply:
Yeah, I agree. Right now, we seem to reuse the existing mock client between the tests. I didn't realize, that the InitMockOvnClients() was not truly giving a new instance, just using the pre-created global instance between tests.